src/objective-c/tests/InteropTestsLocalSSL.m

```diff
@@ -57,4 +57,15 @@ - (void)setUp {
   [super setUp];
 }
 
+- (void)testExceptions {
```

Comment:
This is an odd place to put this test. Maybe create a GRPCCallUnitTests file and put there all tests that don't require a server?